src/models/Offer.js

```diff
@@ -116,7 +116,7 @@ function validateEndDate(value) {
 
 // jobMaxDuration must be larger than jobMinDuration
 function validateJobMaxDuration(value) {
-    return value >= this.jobMinDuration;
+    return value > this.jobMinDuration;
```

Comment:
I changed this to be coherent with the validators on the router layer. Before this change, an offer with equal jobMaxDuration and jobMinDuration would be accepted in the model and rejected in the router

Reply:
We should keep the = on both sides, since that's a special case for offers that have a strict duration

e.g., for offers that strictly last 3 months, both `jobMinDuration` and `jobMaxDuration` will be 3